agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4388,13 +4388,6 @@ bool slave_thread::handle_autoconfiguration_wsc(Socket *sd, ieee1905_1::CmduMess
                    << "     bssid: " << bssid << std::endl
                    << "     authentication_type: " << authentication_type << std::endl
                    << "     encryption_type: " << encryption_type << std::endl;
-
-        if (!manufacturer.compare("Intel")) {
```

Comment:
Can we add the manufacturer print in the above log? will be useful for future debugging.

Reply:
It is already there.